.github/workflows/lint.yml

```diff
@@ -0,0 +1,21 @@
+name: Lint
+on:
+  push:
+    branches:
+      - main
+      - release/*
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  copyright:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Copywrite
+        uses: hashicorp/setup-copywrite@867a1a2a064a0626db322392806428f7dc59cb3e # v1.1.2
```

Comment:
I thought we didn't need to pin `hashicorp/*` namespaced actions? It would be nice to be able to automatically pick up updates to the action without having to deal with a dependabot update